discovery_child_development/getters/taxonomy_classifier.py

```diff
@@ -0,0 +1,244 @@
+from datasets import Dataset
+import pandas as pd
+from typing import Any, Dict, List, Optional
+
+from nesta_ds_utils.loading_saving import S3 as nesta_s3
+
+from discovery_child_development import (
+    PROJECT_DIR,
+    logging,
+    config,
+    S3_BUCKET,
+    taxonomy_config,
+)
+from discovery_child_development.utils.utils import create_directory_if_not_exists
+from discovery_child_development.utils import jsonl_utils as jsonl
+
+# get_labelling_sample
+S3_LABELLING_DATA = (
+    "data/labels/taxonomy_classifier/training_validation_data_patents_openalex.jsonl"
+)
+LOCAL_PATH = PROJECT_DIR / "inputs/data/labelling/taxonomy/input"
+LOCAL_FILE = f"{LOCAL_PATH}/training_validation_data_patents_openalex.jsonl"
+create_directory_if_not_exists(LOCAL_PATH)
+
+# get_gpt_labelled_sample
+GPT_LABELLED_DATA = "data/labels/taxonomy_classifier/labelled_with_gpt/training_validation_data_patents_openalex_GPT_LABELLED.parquet"
```

Comment:
Hey, would this be your previously labelled data or the newly generated using the "v2" pipeline?